cmd/client.go

```diff
@@ -59,14 +72,51 @@ var cliCmd = &cobra.Command{
 			}
 			startTs = oracle.ComposeTS(ts, logical)
 		}
+
+		cfg := new(config)
+		if len(configFile) > 0 {
+			if err := strictDecodeFile(configFile, "cdc", cfg); err != nil {
+				return err
+			}
+		}
+
 		detail := &model.ChangeFeedDetail{
-			SinkURI:    sinkURI,
-			Opts:       make(map[string]string),
-			CreateTime: time.Now(),
-			StartTs:    startTs,
-			TargetTs:   targetTs,
+			SinkURI:             sinkURI,
+			Opts:                make(map[string]string),
+			CreateTime:          time.Now(),
+			StartTs:             startTs,
+			TargetTs:            targetTs,
+			FilterRules:         cfg.FilterRules,
+			FilterCaseSensitive: cfg.FilterCaseSensitive,
 		}
-		fmt.Printf("create changefeed ID: %s detail %+v\n", id, detail)
+		d, err := detail.Marshal()
+		if err != nil {
+			return err
+		}
+		fmt.Printf("create changefeed ID: %s detail %s\n", id, d)
 		return kv.SaveChangeFeedDetail(context.Background(), cli, detail, id)
 	},
 }
+
+// strictDecodeFile decodes the toml file strictly. If any item in confFile file is not mapped
+// into the Config struct, issue an error and stop the server from starting.
+func strictDecodeFile(path, component string, cfg interface{}) error {
```

Comment:
Is it better to put this function in a general lib and set it public.
Also recommend adding a unit test and a config sample file

Reply:
Unit tests and example file `cmd/cdc.toml` added.

Since no other places use this function now, I think it's better if we keep it close to where it's used. We can easily move it to a general package later if other modules need it.